Backpack-SwiftUI/FlowStackView/Classes/BPKFlowStackView.swift

```diff
@@ -92,8 +92,10 @@ public struct BPKFlowStackView<Data: Collection, Content: View>: View where Data
             let doesNotFitInCurrentRow = elementSize.width + spacing.width > remainingWidth
             
             if doesNotFitInCurrentRow { // new row
-                remainingWidth = availableWidth
-                currentRow += 1
+                if rows.indices.contains(currentRow) {
```

Comment:
If the current row does not have any elements yet, `currentRow` does not need to be + 1